### ISSUE TITLE
match reports: could not find the newly generated certificate installed

### ISSUE THREAD

**Issue by TienNM** [fastlane/fastlane#4318]
### Issue Description

I can code sign using Xcode (using the fix issue button) but when I use,

match nuke development
match development -a com.xxx.appname,

I get an error: 
Couldn't find a valid code signing identity in the git repo for development... creating one for you now
...
 Could not find the newly generated certificate installed

The full output is shown below

##### Complete output when running fastlane, including the stack trace and command used

```
sh-3.2# match development

+-----------------------+----------------------------------------------------------+
|                             Summary for match 0.4.0                              |
+-----------------------+----------------------------------------------------------+
| git_url               | http://xxx@xxx.com:7990/scm/PROF/profiles.git |
| type                  | development                                              |
| username              | xxx@xxx.com                                |
| keychain_name         | login.keychain                                           |
| readonly              | false                                                    |
| verbose               | false                                                    |
| force                 | false                                                    |
| shallow_clone         | false                                                    |
| force_for_new_devices | false                                                    |
| skip_docs             | false                                                    |
+-----------------------+----------------------------------------------------------+

[16:38:37]: Cloning remote git repo...
[16:38:43]: 🔓  Successfully decrypted certificates repo
[16:38:43]: Verifying that the certificate and profile are still valid on the Dev Portal...
[16:38:47]: To not be asked about this value, you can specify it using 'app_identifier'
The bundle identifier of your app: com.bluemangoo.ifretlessbass
[16:38:59]: Couldn't find a valid code signing identity in the git repo for development... creating one for you now

+---------------+--------------------------------------------+
|                   Summary for cert 1.4.0                   |
+---------------+--------------------------------------------+
| development   | true                                       |
| force         | true                                       |
| username      | xxx@xxx.com                  |
| keychain_path | /var/root/Library/Keychains/login.keychain |
+---------------+--------------------------------------------+

[16:38:59]: Starting login with user 'xxx@xxx.com'
[16:39:06]: Successfully logged in
[16:39:11]: $ security import /var/folders/zz/zxxx_n0000000000000/T/dxxxnitux/certs/development/xxx.p12 -k '/var/root/Library/Keychains/login.keychain' -T /usr/bin/codesign -T /usr/bin/security
[16:39:11]: ▸ 1 key imported.
[16:39:11]: $ security import /var/folders/zz/zyxxx000000/T/dxxxitux/certs/development/xxxxxW.cer -k '/var/root/Library/Keychains/login.keychain' -T /usr/bin/codesign -T /usr/bin/security
[16:39:11]: ▸ 1 certificate imported.
[16:39:11]: Successfully generated xxxxxxxxxxx which was imported to the local machine.
[16:39:12]: There are no local code signing identities found. You can run `security find-identity -v -p codesigning` to get this output. This Stack Overflow thread has more information: http://stackoverflow.com/q/35390072/774. (Check in Keychain Access for an expired WWDR certificate: http://stackoverflow.com/a/35409835/774 has more info.)
[16:39:12]: Verifying the certificated is properly installed locally...
[16:39:12]: 🔒  Successfully encrypted certificates repo

[!] Could not find the newly generated certificate installed
```

### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Matchfile:
```ruby
git_url "http://xxx@xxx.com:7990/scm/PROF/profiles.git"

type "development" # The default type, can be: appstore, adhoc or development

#app_identifier "xxx.xxx.appname" 
username "xxx@xxx.com" 
```

### Environment

fastlane version (run `fastlane -v`): 1.81.0
match version: 0.4.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? no

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? no

**Comment by sirhans** [comment]
Hi, I'm from the same team as the original poster.  We just solved this problem.  Here is the solution:

On another team member's machine, where match was working properly:

```
match nuke
match development
```

On the machine where match was not working:

```
git credentials-osxkeychain erase
host=xxx.com
```

We had entered a wrong password and git saved it.  We logged into our git server so many times with wrong password that we got locked out and required to enter a CAPTCHA.  So we logged in to our server (Atlassian Stash), confirmed our password for the account we were using, and deleted the old git credentials as shown above.  Then all was well.
